pkgs/by-name/si/sieve-editor-gui/package.nix

```diff
@@ -1,42 +1,71 @@
 {
   lib,
+  stdenvNoCC,
```

Comment:
It is safe to use stdenv here and makes no difference. If ever stdenv is added, not both should exist.